### PR TITLE
Change visibility of some methods and variables in Plugins

### PR DESCRIPTION
**Pull request by arthursz**
Goal
---
Change visibility of some methods and variables in Plugins

How to test
---
1 - Run Unit tests
2 - Open sample app
3 - Check if video everything is working correctly, play/pause the video, perform seek, wait for it to finish and play it again, go fullscreen and back to portrait

### CODE REVIEW

File: clappr/src/main/kotlin/io/clappr/player/plugin/control/SeekbarPlugin.kt [hunk 1]
```diff
@@ -31,10 +31,9 @@ open class SeekbarPlugin(core: Core) : MediaControl.Plugin(core) {
 
     open val scrubberView by lazy { view.findViewById(R.id.scrubber) as View }
 
-    open var dragging = false
-
-    internal val playbackListenerIds = mutableListOf<String>()
+    var dragging = false
```
Comment (UelitonFreitas) [inline clappr/src/main/kotlin/io/clappr/player/plugin/control/SeekbarPlugin.kt]: I suggest to make this variable `protected`. We can use `updateDrag()` method to set `dragging` state on tests.

File: clappr/src/main/kotlin/io/clappr/player/plugin/PosterPlugin.kt [hunk 2]
```diff
@@ -22,7 +22,7 @@ class PosterPlugin(container: Container): UIContainerPlugin(container) {
 
     private val imageView = ImageView(context)
 
-    var posterImageUrl: String? = null
+    internal var posterImageUrl: String? = null
```
[the remark this answers is not in the document]
Comment (arthursz) [reply clappr/src/main/kotlin/io/clappr/player/plugin/PosterPlugin.kt]: Done ;)

File: clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt [hunk 3]
```diff
@@ -48,31 +48,31 @@ open class MediaControl(core: Core) : UICorePlugin(core) {
 
     private val handler = Handler()
 
-    open var lastInteractionTime = 0L
+    internal var lastInteractionTime = 0L
```
[the remark this answers is not in the document]
Comment (arthursz) [reply clappr/src/main/kotlin/io/clappr/player/plugin/control/MediaControl.kt]: Done ;)

File: clappr/src/main/kotlin/io/clappr/player/plugin/control/SeekbarPlugin.kt (hunk 1 above)
Comment (arthursz) [reply clappr/src/main/kotlin/io/clappr/player/plugin/control/SeekbarPlugin.kt]: Done ;)

**Pull request merged**